lib/ingest_api.ex

```diff
@@ -47,22 +49,22 @@ defmodule Logger.Backend.Humio.IngestApi do
   def format_message(
         %{message: msg, level: level, timestamp: ts, metadata: md},
         format,
+        Logger.Formatter,
         metadata_keys
       ) do
-    msg
-    |> IO.chardata_to_string()
-    |> String.split("\n")
-    |> filter_empty_strings
-    |> Enum.map(&format_event(level, &1, ts, md, format, metadata_keys))
-    |> Enum.join("")
-  end
-
-  defp format_event(level, msg, ts, md, format, metadata_keys) do
     Logger.Formatter.format(format, level, msg, ts, take_metadata(md, metadata_keys))
+    |> IO.chardata_to_string()
```

Comment:
nit pipe chains should start with a raw value
```suggestion
    format
    |> Logger.Formatter.format(level, msg, ts, take_metadata(md, metadata_keys)) 
    |> IO.chardata_to_string()
```